### DISCUSSION TITLE
Change naming of importing beatmaps to beatmapsets

### DISCUSSION THREAD

**Discussion by lassedds** [ppy/osu#26933] [Ideas]
When importing beatmaps it should say "Beatmapsets" as this is more correct for players and less confusing, for example the importer says "Importing 8,000 maps" but when I go in the play menu I have 12,000 maps which is probably confusing and misleading

So for clarification I think it should be called "beatmapsets" instead

**Comment by Joehuu** [comment]
>When importing beatmaps it should say "Beatmapsets" as this is more correct for players 

Not what's said in the [wiki page](https://osu.ppy.sh/wiki/en/Beatmap):
> A beatmap (sometimes called beatmapset)

Emphasis on "sometimes". Should be changed if it's "more correct".

---

"Beatmapset" or "beatmap set" is supposed to be an internal (in code) term, see table:

| Internal (in code) | Wiki term (user shown) |
| --- | --- |
| Beatmap | [Difficulty](https://osu.ppy.sh/wiki/en/Beatmap/Difficulty) |
| Beatmap set | [Beatmap](https://osu.ppy.sh/wiki/en/Beatmap) |

Not sure why it's not the same. The terminology gets mixed up, see https://github.com/ppy/osu/pull/23000 for example. The solution for that example in the end was making the song select term ambiguous with "matches", but that can be interpreted as any of the terms in the table.
